src/models/StateType.ts

```diff
@@ -5,6 +5,7 @@ export const stateType = [
   'pending',
   'failed',
   'cancelled',
+  'cancelling',
```

Comment:
I believe there is a utility function for mapping state names to state types that probably needs to be updated to reflect this addition.